Sources/MarathonCore/ScriptManager.swift

```diff
@@ -94,7 +94,7 @@ internal final class ScriptManager {
 
     // MARK: - API
 
-    func script(at path: String) throws -> Script {
+    func script(at path: String) throws -> Script {  // TODO: change param to `atPath`
```

Comment:
Good point 👍 I think that's out of scope for this PR though, do you want to create a new issue to track the renaming of that method? 🙂